services/summaries/summaries.py

```diff
@@ -58,6 +58,33 @@ def ignore_domain(domain):
     )
 
 
+def get_domain_negative_findings(db, domain_id):
+    cursor = db.aql.execute(
+        """
+            LET emailTags = (
+                FOR dnsScan, dnsE IN 1 OUTBOUND @domain_id domainsDNS
+                    SORT dnsScan.timestamp DESC
+                    LIMIT 1
+                    RETURN FLATTEN([dnsScan.dmarc.negativeTags, dnsScan.dkim.negativeTags, dnsScan.spf.negativeTags])
+            )[0]
+            LET webTags = (
+                FOR web, webE IN 1 OUTBOUND @domain_id domainsWeb
+                    SORT web.timestamp DESC
+                    LIMIT 1
+                    FOR webScan, webScanE IN 1 OUTBOUND web webToWebScans
+                        RETURN FLATTEN([webScan.results.tlsResult.negativeTags, webScan.results.connectionResults.negativeTags])
```

Comment:
This will count web tags multiple times per domain if the endpoints each have the tag. Which is fine if we want to display it that way. 
```
https14: 321 domains
```
 vs 
```
https14: 400 web servers/endpoints/whateverweshouldcallit
```

Reply:
I think we do, since it's a summary of the total amount of negative tags. Good catch - I'll work on the wording